<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabshow</class>
 <widget class="QDialog" name="tabshow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>881</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>881</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>881</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>670</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
